video-course/src/app/authentication.service.ts

```diff
@@ -0,0 +1,25 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AuthenticationService {
+
+  constructor() { }
+
+  logIn() {
+    console.log('Login was successful');
+  }
+
+  logOff() {
+    console.log('Logoff was clicked');
+  }
+
+  isAuthenticated() {
+
+  }
+
+  getUserInfo() {
+
+  }
+}
```

Comment:
this should be part of auth / logging module
this method: isAuthenticated() { } you can use to define what is displayed in header (login or logout)
for now you can hardcode return true / false